File: app/src/main/java/com/tomaszrykala/dorefindmi/things/supplier/digidisplay/DigiDisplaySupplier.java
package com.tomaszrykala.dorefindmi.things.supplier.digidisplay;

import com.tomaszrykala.dorefindmi.things.supplier.BaseSupplier;

public interface DigiDisplaySupplier extends BaseSupplier {

    void display(double n);

    void display(String s);
}

File: app/src/main/java/com/tomaszrykala/dorefindmi/things/supplier/digidisplay/DigiDisplaySupplier.kt
package com.tomaszrykala.dorefindmi.things.supplier.digidisplay

import com.tomaszrykala.dorefindmi.things.supplier.BaseSupplier

interface DigiDisplaySupplier : BaseSupplier {

    fun display(n: Double)

    fun display(s: String)
}
